Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1004,109 +1004,116 @@ ScanIntelProcessor (
   }
 
   //
-  // TODO: this may not be accurate on some older processors.
+  // When the CPU is virtualized and cpuid invtsc is enabled, then we already get the information
+  // we want outside the function, skip anyway.
+  // Things maybe different in other hypervisors, but should work with QEMU/VMWare now
```

Comment:
`Things may be different in other hypervisors, but should work with QEMU/VMWare for now.`